.github/pull_request_template.md

```diff
@@ -3,7 +3,7 @@ Please complete the following checklist if your PR is adding new link to the lis
 - [ ] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
 - [ ] This PR does not introduce duplicates to the list
 - [ ] The link is added at the bottom of the list category
-- [ ] Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
+- [ ] The suggested repository is maintained, have an active community, can mentor new contributors and have issues with the suggested labels.
```

Comment:
`labels` should be singular - there is only one label that can be mentioned for each repository in the list (at least at the moment)